readthedocs/profiles/urls/private.py

```diff
@@ -31,6 +30,11 @@
         views.AccountDelete.as_view(),
         name='delete_account',
     ),
+    url(
+        r'security-log/',
```

Comment:
Should this be called `audit-log` instead, to match branding?

Reply:
GitHub calls them security log, but I'm fine renaming it.